src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java

```diff
@@ -72,6 +72,27 @@ public void childThreadTest() {
         assert(childThread.ftzValue == false);
     }
 
+    @Test(expectedExceptions = {NullPointerException.class})
+    public void nullfileNameTest() {
+
+        IntelGKLUtils utils = new IntelGKLUtils();
+        String filename = IntelGKLUtils.pathToTestResource(null);
```

Comment:
Suggestion - since we are not validating/asserting return value from this call due to exception thrown, may not be needed to store it into `filename` variable. Same applies to other tests too.  For example 
```
IntelGKLUtils utils = new IntelGKLUtils();
IntelGKLUtils.pathToTestResource(null);
```